packages/linux/package.mk

```diff
@@ -89,6 +89,12 @@ post_patch() {
     sed -i -e "s|^CONFIG_ISCSI_IBFT_FIND=.*$|# CONFIG_ISCSI_IBFT_FIND is not set|" $PKG_BUILD/.config
     sed -i -e "s|^CONFIG_ISCSI_IBFT=.*$|# CONFIG_ISCSI_IBFT is not set|" $PKG_BUILD/.config
   fi
+
+  # disable lima/panfrost if libmali is configured
+  if [ "$OPENGLES" = libmali ]; then
```

Comment:
Can you double quote both sides please? :)

Reply:
done